code/modules/materials/definitions/solids/materials_solid_fission.dm

```diff
@@ -0,0 +1,81 @@
+/decl/material/solid/metal/depleted_uranium
+	name = "depleted uranium"
+	lore_text = "Uranium that does not posess a significant amount of radioactive isotopes. Extremely dense, and can be enriched to produce more fission fuel."
+	mechanics_text = "Depleted uranium can be enriched in fission reactors for use as fuel."
+	taste_description = "the outside of a reactor"
+	flags = MAT_FLAG_FISSIBLE
+	icon_base = 'icons/turf/walls/stone.dmi'
+	table_icon_base = "stone"
+	icon_reinf = 'icons/turf/walls/reinforced_stone.dmi'
+	value = 1.5
+
+	neutron_cross_section = 5
+	neutron_interactions = list(INTERACTION_ABSORPTION = 2500, INTERACTION_FISSION = 4500)
+	absorption_products = list(
+		/decl/material/solid/metal/plutonium = 0.8
+	)
+	fission_products = list(
+		/decl/material/solid/metal/fission_byproduct = 0.8
+	)
+	fission_heat = 35000
+	fission_energy = 4000
+	neutron_absorption = 4
+	
+
+/decl/material/solid/metal/neptunium // Np-237.
+	name = "neptunium"
+	lore_text = "A byproduct of uranium undergoing beta decay. Extremely radioactive, can be used as fission fuel, with difficulty."
+	mechanics_text = "Neptunium can be used as fuel in fission reactors at high neutron energies."
+	taste_description = "lemon juice and hot concrete."
+	flags = MAT_FLAG_FISSIBLE
+	radioactivity = 30
+	icon_base = 'icons/turf/walls/stone.dmi'
+	table_icon_base = "stone"
+	icon_reinf = 'icons/turf/walls/reinforced_stone.dmi'
+	color = "#404c53"
+	value = 0.5
+
+	neutron_cross_section = 4 // Difficult to use as fuel.
+	neutron_interactions = list(INTERACTION_FISSION = 3000)
+	fission_products = list(
+		/decl/material/solid/metal/fission_byproduct = 0.2
+	)
+
+/decl/material/solid/metal/plutonium
+	name = "plutonium"
+	lore_text = "A mundane silver-grey metal that is highly fissible. Often used as fuel in nuclear fission reactors and weapons."
+	mechanics_text = "Plutonium can be used as fuel in fission reactors."
+	taste_description = "nuclear fallout"
+	flags = MAT_FLAG_FISSIBLE
+	icon_base = 'icons/turf/walls/stone.dmi'
+	table_icon_base = "stone"
+	icon_reinf = 'icons/turf/walls/reinforced_stone.dmi'
+	color = "#b5c5a2"
+	value = 3
+
+	neutron_cross_section = 10
+	neutron_interactions = list(INTERACTION_FISSION = 1000)
```

Comment:
Pls pick a style for lists, you have several multi-key lists on single lines and then random single-key lists on multiple lines.